src/fsharp/TypedTreeOps.fs

```diff
@@ -8369,9 +8403,13 @@ let MemberIsCompiledAsInstance g parent isExtensionMember (membInfo: ValMemberIn
 
 let isSealedTy g ty =
     let ty = stripTyEqnsAndMeasureEqns g ty
-    not (isRefTy g ty) ||
-    isUnitTy g ty || 
-    isArrayTy g ty || 
+    let isRefTy' = isRefTy g ty
```

Comment:
Remove this change I don't thinkn it does anything?

Reply:
Only made this change to inspect each of the statement when debugging. 👍🏼